tests/unit/runtime/half_precision/test_bf16.py

```diff
@@ -27,6 +27,7 @@ def test(self, zero_stage=2, use_cpu_offload=False):
             pytest.skip("cpu-adam is not compatible")
 
         config_dict = {
+            "train_batch_size": 4,
```

Comment:
It is more robust to use `"train_micro_batch_size_per_gpu": 1`

Reply:
Should I do this for all tests in that file?

Reply:
Yes, that would be great. Thanks!